### PR TITLE
fix: FilePickerSelectedFilesArray.Count shouldn't have a throwing setter #6885

### LINKED ISSUE #6885: `FilePickerSelectedFilesArray.Count` shouldn't have a throwing setter
### Current behavior

FilePickerSelectedFilesArray.Count has a setter that always throws

### Expected behavior

It shouldn't have a setter to align with UWP.

### How to reproduce it (as minimally and precisely as possible)

_No response_

### Workaround

_No response_

### Works on UWP/WinUI

Yes

### Environment

_No response_

### NuGet package version(s)

_No response_

### Affected platforms

_No response_

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by mcNets**
GitHub Issue (If applicable): closes #6885

https://github.com/unoplatform/uno/issues/6885

## PR Type

What kind of change does this PR introduce?

- Feature

## What is the current behavior?

It's thowing an InvalidOperationException: 

`set => throw new InvalidOperationException();`

## What is the new behavior?

There is no setter.

### <samp>🤖 Generated by Copilot at 6fecc1d</samp>

Fix a bug in `FileOpenPicker` on UWP by commenting out the `Count` setter in `FilePickerSelectedFilesArray`. This prevents an exception when the array is modified by the `IList<T>` interface.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [X] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [X] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

https://github.com/unoplatform/uno/issues/6885

### CODE REVIEW

File: src/Uno.UWP/Storage/Pickers/FilePickerSelectedFilesArray.cs [hunk 1]
```diff
@@ -35,7 +35,7 @@ internal FilePickerSelectedFilesArray(StorageFile[] items)
 		public int Count
 		{
 			get => _items.Count;
-			set => throw new InvalidOperationException();
+			//set => throw new InvalidOperationException();
```
Comment (jeromelaban) [inline src/Uno.UWP/Storage/Pickers/FilePickerSelectedFilesArray.cs:38]: It's probably a breaking change to remove this particular setter. Is it available on Windows? If not, it's likely to be an acceptable breaking change for 5.0.

Comment (mcNets) [reply src/Uno.UWP/Storage/Pickers/FilePickerSelectedFilesArray.cs:38]: According to [this](https://github.com/unoplatform/uno/issues/6885#issuecomment-911658880) comment it should be acceptable.

Comment (jeromelaban) [reply src/Uno.UWP/Storage/Pickers/FilePickerSelectedFilesArray.cs:38]: Right! You'll likely need to add a exclusion here: https://github.com/unoplatform/uno/blob/master/build/PackageDiffIgnore.xml

Comment (mcNets) [reply src/Uno.UWP/Storage/Pickers/FilePickerSelectedFilesArray.cs:38]: Done

**Review by jeromelaban — APPROVED** [review]

File: src/Uno.UWP/Storage/Pickers/FilePickerSelectedFilesArray.cs (hunk 1 above)
Comment (jeromelaban) [reply src/Uno.UWP/Storage/Pickers/FilePickerSelectedFilesArray.cs:38]: You can also remove the commented line, it's not useful anymore. Thanks!

Comment (mcNets) [reply src/Uno.UWP/Storage/Pickers/FilePickerSelectedFilesArray.cs]: Removed.

**Review by jeromelaban — APPROVED** [review]

File: src/Uno.UWP/Storage/Pickers/FilePickerSelectedFilesArray.cs (hunk 1 above)
Comment (mcNets) [reply src/Uno.UWP/Storage/Pickers/FilePickerSelectedFilesArray.cs]: @jeromelaban can this conversation be resolved?

**Pull request merged**